js/sia.js

```diff
@@ -189,7 +207,7 @@ function SiadWrapper () {
    * @returns {object} siad configuration object
    */
   function configure (settings, callback) {
-    for (var key in settings) {
+    for (let key in settings) {
```

Comment:
Yup, it's a minor thing but we ought to use ES5 and even ES6 conventions more. They're powerful.